### ISSUE TITLE
the lake ice surface temperature is wrong?

### ISSUE THREAD

**Issue by lzg0620** [AquaticEcoDynamics/GLM#4]
@lawinslow  In recent days, I  simulated a Tibetan Plateau lake using GLM model, where the observed the lake ice surface temperature was very low (<-10°C). However, the simulated lake surface temperature (lake.csv file) is still higher than 0°C during the freeze period (the depth of ice is larger than 0.4 m). Moreover, I also found the ice surface temperature was also higher than 0°C  in your coldlake examples. I feel very confused I further checked the glm_surface.c code file, and found the ice surface temperature is approximately equal to zero except for the ice albedo calculation section. In the <GLM Model Overview and User Information  v 2.0 β - Oct 2014>，there was a similar description (T0 is determined using a bilinear iteration until surface heat fluxes are balanced (i.e. φ0(T0) = -φnet (T0)) and T0 is stable ( ± 0.001 oC).).

Is right what I understand? I just want to get a correct ice surface temperature output.
Thank you.

**Comment by lawinslow** [comment]
@lzg0620 what source of ice surface temperature data are you using? I don't think it is being output anywhere. *Lake surface* temperature, because it is the first liquid layer of the lake, could certainly be above 0°C.

**Comment by lawinslow** [comment]
Yeah, stuck a line in the code. Definitely seeing negative surface temps
```
Ice temperature -5.664062
Ice temperature -6.250000
Ice temperature -6.835938
Ice temperature -7.617188
Ice temperature -7.812500
```

Do you want a way to get access to the ice surface temperatures in the GLM output? We might be able to add it.

**Comment by lzg0620** [comment]
@lawinslow Thanks for your kind reply. Sorry, that's wrong what I understand. It is very good if you can add the ice surface temperature to the output file, because sometimes many users need to verify the reliability of the simulation results using the surface temperature based on the remote sensing product (eg. MODIS). At that time, the ice surface temperature is important, and the  first liquid layer of the lake is difficult to directly verify.
Thanks again.
